src/chain_spec.rs

```diff
@@ -157,6 +159,13 @@ impl Alternative {
     }
 }
 
+fn polymath_props() -> Properties {
+    json!({"tokenDecimals": 6, "tokenSymbol": "POLY" })
```

Comment:
@JesseAbram We are avoiding the use of any code that could generate `panic` (safety reason), like `unwrap`.
This is not the case, but it is going to be easier to find `panic` functions if we limit their use.
In this PR, you could just use something like: 
```
fn polymath_props() -> Option<Properties> {
    json!({"tokenDecimals": 6, "tokenSymbol": "POLY" }).as_object().cloned()
}
```

Reply:
Just for my own personal understanding. 

unwrap would handle a null value as a panic, which is against your coding standards...... which ya running a node makes sense. However, in this scenario the data would be hardcoded and would be compiled pre running the node into a chianspec and not able to be called again. 

Totally get why the change was made as coding standards, just personally trying to get a better understanding.

Reply:
You're right this code is **completely safe**. :+1: 
It is just a facility to keep any `unwrap` (or any other `panic` function) at minimum. We could use a simple `grep` to count the number of `unwrap` and add this as a kind of "safety KPI".
We are using `unwrap` only in _unit test_, and removing them at **production code**.

Less `panic` fn (or `unsafe` fn) means safer code :smile:

Reply:
Makes sense, Thank you!